### ISSUE TITLE
Need help downloading Scratchpad w/DwC Archiver 

### ISSUE THREAD

**Issue by jpsullivan65** [NaturalHistoryMuseum/scratchpads2#6649]
(1) I followed the instructions for downloading my Scratchpad as a DarwinCore archive. I was assuming this would download all content into a massive file(s). All I got was a folder of tiny .xml and .csv files, see screencap attached.  Is this what's supposed to happen? Where in these files is my classification?  If this worked as it should, what is one supposed to do now?

(2) I've been asked if a Scratchpad can be downloaded as an SQL file.  Can it?

![Screenshot 2023-04-27 at 11 06 41 AM](https://user-images.githubusercontent.com/21112349/234909744-85ab6e83-8ad0-41b1-b8d6-08f4005011ac.jpg)

**Comment by benscott** [comment]
Yes, you can download a full archive of your site, including SQL database and all files.  Instructions are available here https://scratchpads.readthedocs.io/en/latest/export/create-backup.html
